src/schema/provenance_schema.yaml

```diff
@@ -874,6 +880,11 @@ ENTITIES:
     excluded_properties_from_public_response:
       - lab_tissue_sample_id
       - submission_id
+      - direct_ancestor:
+        - lab_donor_id
+        - submission_id
+        - label
+        - lab_tissue_sample_id
```

Comment:
- Similarly, add comments to indicate which fields belong to which entity type for clarity. 
- Add `metadata.lab_id` under this `direct_ancestor` section, for when the sample's parent is another sample/organ.

Reply:
done